cross/ffmpeg6/Makefile

```diff
@@ -10,13 +10,16 @@ COMMENT  = FFmpeg is a complete, cross-platform solution to record, convert and
 LICENSE  = GPLv2
 
 CONFIGURE_ARGS  = --target-os=linux --cross-prefix=$(TC_PATH)$(TC_PREFIX) --prefix=$(INSTALL_PREFIX)
-CONFIGURE_ARGS += --extra-cflags="-I$(WORK_DIR)/install$(INSTALL_PREFIX)/include"
-CONFIGURE_ARGS += --extra-ldflags="-L$(WORK_DIR)/install$(INSTALL_PREFIX)/lib"
-CONFIGURE_ARGS += --extra-libs="-lxml2 -ldl -lm" --pkg-config=/usr/bin/pkg-config --ranlib=$(RANLIB)
```

Comment:
This code dates way-back, before I started maintaining ffmpeg.  Why is only pkg-config related code kept?